### ISSUE TITLE
How to name comment

### ISSUE THREAD

**Issue by masatoota22** [openPMD/openPMD-api#329]
I can make a comment using set_comment in Value. But the name is always "comment" in Name. How can I name it as I  want?

**Comment by anokfireball** [comment]
Hi, @masatoota22.
The [```Attributable::setComment(std::string const& value)```](https://www.openpmd.org/openPMD-api/classopen_p_m_d_1_1_attributable.html#aac0a5610fa3879e26d56c8f38123e3f8) call corresponds to the [comment](https://github.com/openPMD/openPMD-standard/blob/latest/STANDARD.md#hierarchy-of-the-data-file) Attribute specified in the openPMD standard. This dictates that the corresponding attribute will have key `comment`.

To write an arbitrary key/value pair as an attribute to a dataset, you can use  [```Attributable::setAttribute(std::string const& key, T&& value)```](https://www.openpmd.org/openPMD-api/classopen_p_m_d_1_1_attributable.html#ad943535647ea6a2cd497b9f10842cef4). An example for how to use it is available in [examples/7_extended_write_serial.cpp](https://github.com/openPMD/openPMD-api/blob/dev/examples/7_extended_write_serial.cpp#L57-L58).

**Comment by masatoota22** [comment]
Thank you for replying. Can you tell me the way in python?

**Comment by anokfireball** [comment]
I'm unsure about the current status concerning Python attributes. The binding is still very much work in progress. @ax3l knows a thing about it or two.

There is currently a PR underway (#314) that tries to replicate `examples/7_extended_write_serial.cpp` in Python. Looking at the code and #330, I suspect it is only partially functional:
https://github.com/openPMD/openPMD-api/blob/3ac2508b8aff25285858af5b9c680c8f0d6c00fa/examples/7_extended_write_serial.py#L32-L36

**Comment by ax3l** [comment]
Hi @masatoota22 , you want to write an attribute with `set_attribute` that is a `string`, right?

This is currently broken in `0.4.0a` and will be fixed in the next days. #330 already implements the fix for operations such as:
```python
series.iteration[1].set_attribute("other_name_than_comment", "and_its_string_value")
```

**Comment by ax3l** [comment]
Hey @masatoota22,

we think the issue you reported is now fixed in openPMD-api 0.5.0a! :)

First of all, sorry for the delay in the bug fix. The underlying type system got a full rework so that python attribute writing also works as expected now.

If you update your conda install to the latest version of openPMD-api, version 0.5.0a, this should be fixed:
```bash
conda update -c conda-forge openpmd-api
```

You can now write any kind of attribute name and type as you planned, e.g.:
```python
series.set_attribute("something_on_the_series", "wohooo")
series.set_attribute("someNumber_on_the_series", 42)
series.iteration[1].set_attribute("other_name_than_comment", "and_its_string_value")
# ...
```

Can you confirm?
